osu.Game.Tests/Skins/LegacySkinDecoderTest.cs

```diff
@@ -13,28 +13,23 @@ namespace osu.Game.Tests.Skins
     [TestFixture]
     public class LegacySkinDecoderTest
     {
-        [TestCase(true)]
-        [TestCase(false)]
-        public void TestDecodeSkinColours(bool hasColours)
```

Comment:
why did you remove the test for no colours?

Reply:
As told in the description of that commit, checked differently with `TestSceneSkinConfigurationLookup.TestEmptyComboColours(false)`

Reply:
no reason to not leave the test here?

Reply:
it'll fail as the combo colours are gonna be empty? unless it's fine for it to be there

Reply:
i really don't want to have to spell it out, but it seems i have to:

update the test asserts to match the *expected* output.